.github/workflows/build-verification.yml

```diff
@@ -70,26 +78,28 @@ jobs:
       - name: Setup ZULU_JDK
         uses: actions/setup-java@v3
         with:
-          java-version: 11.0.12
+          java-version:  ${{ matrix.java.jdkImageIssue }}
           distribution: 'zulu'
       - name: Set ZULU_JDK environment variable
         run: |
           echo "ZULU_JDK=$JAVA_HOME" >> $GITHUB_ENV
       - name: Setup ZULU_ALT_JDK
         uses: actions/setup-java@v3
         with:
-          java-version: 11.0.13
+          java-version: ${{ matrix.java.jdkImageIssueAlt }}
           distribution: 'zulu'
       - name: Set ZULU_ALT_JDK environment variable
         run: |
           echo "ZULU_ALT_JDK=$JAVA_HOME" >> $GITHUB_ENV
       - name: Set up Java
         uses: actions/setup-java@v3
         with:
-          java-version: 11
+          java-version: ${{ matrix.java.jdk }}
           distribution: 'adopt'
       - name: Set up Gradle
         uses: gradle/gradle-build-action@v2
+      - name: Update libncurses5
+        run: sudo apt update && sudo apt-get install libncurses5
```

Comment:
required to avoid https://ge.solutions-team.gradle.com/s/3iwp5kve57n54/tests/:testAndroid7_2_2/org.gradle.android.CompileLibraryResourcesWorkaroundTest/does%20not%20warn%20about%20experimental%20flags%20when%20applied%20from%20a%20kotlin%20script%20plugin?top-execution=1